fastembed/text/jina_onnx_embedding.py

```diff
@@ -24,6 +24,14 @@
         "sources": {"hf": "xenova/jina-embeddings-v2-small-en"},
         "model_file": "onnx/model.onnx",
     },
+    {
+        "model": "jinaai/jina-embeddings-v2-base-de",
+        "dim": 768,
+        "description": "German embedding model supporting 8192 sequence length",
+        "size_in_GB": 0.16,
```

Comment:
fp16 model has a size of 0.32 GB, 0.16 is the size of the quantized model

```suggestion
        "size_in_GB": 0.32,
```